ocf_datapipes/load/pv/utils.py

```diff
@@ -111,3 +112,38 @@ def encode_label(indexes: List[str], label: str) -> List[str]:
     new_index = [int(int(col) * 10 + label_index) for col in indexes]
 
     return new_index
+
+
+def xr_to_df(pv_power: xr.Dataset, ssid: str, date_oi: str) -> pd.DataFrame():
+    """Function that gives a pv system df of a single day with its output
+
+    Converts xarray dataset into a pandas dataframe,
+    and its values for a single pv systme and a single day
+
+    Args
+        pv_power: pv.netcdf file which contains system is and datetime
+                    with corresponding pv output values
+        ssid: A single ID of the pv system
+        date_oi: Date of Interest
+    """
+    date_1 = datetime.strptime(date_oi, "%Y-%m-%d")
```

Comment:
Maybe its more readable to just pass `date_oi` in this function as a `datetime.datetime` object instead of doing the conversion in the function itself? That way you don't get some errors if your date string is in the wrong format etc.